python/batch-predictor/docker/app.Dockerfile

```diff
@@ -20,6 +20,6 @@ FROM ${BASE_IMAGE}
 ARG MODEL_URL
 RUN gsutil cp -r ${MODEL_URL} .
 RUN /bin/bash -c ". activate merlin-model && \
-    sed -i 's/pip$/pip=20.2.4/' /model/conda.yaml && \
+    sed -i 's/pip$/pip=22.2.2/' /model/conda.yaml && \
```

Comment:
Is there any reason to upgrade pip version too? This was forced to allow dependency conflict which sometimes happen during model building. Without this user's PyFunc and batch predictor model might not be able to be built.